exonum/src/api/backends/actix.rs

```diff
@@ -133,7 +133,13 @@ impl ResponseError for ApiError {
             ApiError::Storage(err) => HttpResponse::InternalServerError().body(err.to_string()),
             ApiError::NotFound(err) => HttpResponse::NotFound().body(err.to_string()),
             ApiError::Unauthorized => HttpResponse::Unauthorized().finish(),
-            ApiError::PayloadTooLarge(err) => HttpResponse::PayloadTooLarge().body(err.to_string()),
+            ApiError::PayloadTooLarge {
+                length_limit,
+                content_length,
+            } => HttpResponse::PayloadTooLarge().body(format!(
+                "Payload too large: Allowed {}, but got to send: {}",
```

Comment:
Micro-nit: I'm not quite sure this is good English... I'd say something like "Payload too large: received $x-byte message, while the allowed message limit is $x bytes" (if necessary, replace "message" with "JSON").